src/chess/piece/Rook.java

```diff
@@ -18,6 +18,7 @@ public boolean isValidMove(Position newPosition) {
     if (!super.isValidMove(newPosition)) {
       return false;
     }
-      return false;
+
+    return true;
```

Comment:
It'd be better if you consider arranging this code with the same part in `Pawn.java`

In `Pawn.java` there is code below

```java
   // ToDo: implement
    return true;
  }
```